improver_tests/acceptance/test_checksums.py

```diff
@@ -30,17 +30,66 @@
 # POSSIBILITY OF SUCH DAMAGE.
 """Bulk checking and updating of checksum file"""
 
+import difflib
+import functools
+import locale
 import os
 import pathlib
+from contextlib import contextmanager
+from typing import Callable, Dict, List
 
 import pytest
 
 from . import acceptance as acc
 
-pytestmark = [pytest.mark.acc, acc.skip_if_kgo_missing]
 
+@contextmanager
+def temporary_sort_locale(collate: str) -> Callable[[str, str], bool]:
+    """
+    Set a temporary locale for sorting.
 
+    Args:
+        collate: LC_COLLATE locale name
+
+    Yields:
+        String comparison function
+    """
+    old = locale.getlocale(locale.LC_COLLATE)
+    try:
+        locale.setlocale(locale.LC_COLLATE, collate)
+        yield locale.strcoll
+    finally:
+        locale.setlocale(locale.LC_COLLATE, old)
+
+
+def checksums_to_text(
+    path_csums: Dict[pathlib.Path, str], sort: bool = False
+) -> List[str]:
+    """
+    Convert checksum dict to sha256sum like format.
+
+    Args:
+        path_csums: per-file checksums
+        sort: apply sorting to file paths
+
+    Returns:
+        List of checksum file lines, no newline at end.
+    """
+    path_csums = {str(path): csum for path, csum in path_csums.items()}
```

Comment:
It might be useful to rename this variable something like `path_csums_str` to make mypy happy. Otherwise it complains in the dict comprehension on this line, and below in the list comprehension. If you go with this approach, you'll need to change any reference to it the lines below in this function.